mappings/net/minecraft/client/model/ModelPart.mapping

```diff
@@ -42,7 +42,7 @@ CLASS net/minecraft/class_630 net/minecraft/client/model/ModelPart
 	METHOD method_22700 getRandomCuboid (Ljava/util/Random;)Lnet/minecraft/class_630$class_628;
 		ARG 1 random
 	METHOD method_22702 renderCuboids (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;IIFFFF)V
-		ARG 1 matrices
+		ARG 1 entry
```

Comment:
`entry` is ambiguous. Leave this `matrices`, an entry of the MatrixStack is a projection matrix along with a model matrix, so matric*es*.

Reply:
Matrices is the matrix stack, though. So you can't call the entries in it matrices too.

The class is also `MatrixStack.Entry`.

Reply:
Yes, you can. The name `matrices` just means there's more than one matrix (whether it's lots of matrices in a stack, or just two matrices for different purposes).

Reply:
`entry` is a bad name, it doesn't say what kind of entry it is. And `matrixStackEntry` is long for nothing, when it could just be `matrices`.